app/components/nypr-o-footer/component.js

```diff
@@ -1,10 +1,14 @@
 import Component from '@ember/component';
 import layout from './template';
-// import { positionBottomCenter } from '@nypr-design-system' ;
+import config from 'nypr-jukeboxes/config/environment';
 
 export default Component.extend({
   layout,
   tagName: '',
+  NEWSLETTER_ENDPOINT: `${config.optInService}/v1/subscribe/mailchimp`,
+  NEWSLETTER_PARAMS: {list: config.jukeboxNewsletter}
+
+
   // calculatePosition(trigger, content) {
   //   return positionBottomCenter(trigger, content, 5);
   // }
```

Comment:
delete commented out code if it is no longer needed.

Reply:
I think this was left by someone else? They may circle back around to it? But deleted either way.